Aidan/dind+bugfix (#233)
latchbio/latch: CHANGELOG.md

### Fixed

> * Create latch config file if it does not exist at registration or develop time

Reviewer: ```suggestion
* Internal state file should be automatically created when running `latch register` and `latch develop`
```

### Added

* `latch init`: Docker in Docker template workflow
* `latch init`: Docker base image

> * Small, medium, and large tasks run with sysbox runtime allowing the execution of system code

Author: Anything that runs in the kernel.

> * Small, medium, and large tasks run with sysbox runtime allowing the execution of system code

Author: AKA not user space

> * Small, medium, and large tasks run with sysbox runtime allowing the execution of system code

Reviewer: ```suggestion
* Small, medium, and large tasks use a [Sysbox runtime](https://github.com/nestybox/sysbox) to run Docker and other system software within task containers
```

> * Small, medium, and large tasks run with sysbox runtime allowing the execution system-level software such as systemd, Docker, Kubernetes, K3s, buildx, legacy apps, and more.

Reviewer 2: ```suggestion
* Small, medium, and large tasks run with sysbox runtime allowing the execution of system-level software such as `systemd`, Docker, Kubernetes, K3s, `buildx`, legacy apps, and more.
```

> * Small, medium, and large tasks use a [Sysbox runtime](https://github.com/nestybox/sysbox) to run Docker and other system software within task containers

Reviewer 2: ```suggestion
* Small, medium, and large tasks use the [Sysbox runtime](https://github.com/nestybox/sysbox) to run Docker and other system software within task containers
```
